extensions/interactions/MultipleChoiceInput/static/multiple_choice_input.css

```diff
@@ -7,9 +7,6 @@
   border: none;
   color: #0D48A1;
   text-align: left;
-  /* This is needed so that images stay bounded by the container in Firefox. */
-  /* This also limits the selection area of the option to the text and radio button. */
-  width: auto;
```

Comment:
I did verify this in firefox and multiple choice interaction worked fine there, even with this removed.